packages/@okta/vuepress-site/docs/guides/oie-upgrade-mfa-enroll-policy/main/index.md

```diff
@@ -1,48 +1,52 @@
 
-For existing apps that manage and use MFA enrollment policies programmatically through the [Policy API](/docs/reference/api/policy/), some development work is required to handle the factor to authenticator conversion schema changes after the org is upgraded to Identity Engine. This guide provides you with key API considerations to upgrade your app for MFA enrollment flows in Identity Engine.
+After upgrading your org to Identity Engine, new authenticator enrollment policies created in the Admin Console are configured using authenticators. MFA enrollment policies created before upgrading to Identity Engine, are still configured with factors. However, if an existing policy was modified and saved in the Admin Console, the factors in that policy are converted to authenticators. This conversion is seamless to admin users that are managing policies in the Admin Console.
 
-## MFA enrollment policy API changes in Identity Engine
+For existing code that manages and uses Classic Engine MFA enrollment policies programmatically through the [Policy API](/docs/reference/api/policy/), some development work is required to handle the factor to authenticator conversion schema changes after the org upgrades to Identity Engine. This guide provides you with key API considerations to upgrade your app for multifactor enrollment flows in Identity Engine.
 
-The following are the main behavior changes to the [MFA enrollment policy](/docs/reference/api/policy/#multifactor-mfa-enrollment-policy) type in Identity Engine:
+## Authenticator enrollment policy API changes in Identity Engine
 
-- The Policy API supports both factors and authenticators schemas in the MFA enrollment policy [settings](/docs/reference/api/policy/#policy-settings-data-2).
-- MFA enrollment policies are created with either factors or authenticators in their settings (they are mutually exclusive).
-- Existing MFA enrollment policies (that is, MFA enrollment policies created before the Identity Engine upgrade) still contain factors in their settings.
-- For existing MFA enrollment policies modified from the Admin Console of an Identity Engine org, existing factors are converted to authenticators in their settings.
+The following are the main behavior changes to the [authenticator enrollment policy](/docs/reference/api/policy/#authenticator-enrollment-policy) in Identity Engine:
+
+- The Policy API supports both factors and authenticators schemas in the authenticator enrollment policy [settings](/docs/reference/api/policy/#policy-settings-data-2).
+- Authenticator enrollment policies are created with either factors or authenticators in their settings (they are mutually exclusive).
+- Existing authenticator enrollment policies (that is, MFA enrollment policies created before the Identity Engine upgrade) still contain factors in their settings.
```

Comment:
should this `(that is, MFA enrollment policies created before the Identity Engine upgrade)` be changed to `(that is, authenticator enrollment policies created before the Identity Engine upgrade)`?